superset/assets/javascripts/dashboard/components/GridLayout.jsx

```diff
@@ -14,117 +12,94 @@ const ResponsiveReactGridLayout = WidthProvider(Responsive);
 
 const propTypes = {
   dashboard: PropTypes.object.isRequired,
+  datasources: PropTypes.object,
+  charts: PropTypes.object.isRequired,
+  filters: PropTypes.object,
+  timeout: PropTypes.number,
+  onChange: PropTypes.func,
+  getFormDataExtra: PropTypes.func,
+  fetchSlice: PropTypes.func,
```

Comment:
a `dashboardActions` object would remove a bunch of boilerplate here I think

Reply:
it's harder to lint actions not being passed with this approach, unless you defined a strict `PropTypes.shape` was defined